### PR TITLE
Add signature for formData types (Fetch API)

### PR DESCRIPTION
**Pull request by alfaslash**
Issue: https://github.com/Microsoft/TypeScript/issues/14378

### CODE REVIEW

**Comment by msftgits** [comment]
Hi, I am closing and re-opening this PR to bump the CLA bot. Sorry for the inconvenience!

File: inputfiles/addedTypes.json [hunk 1]
```diff
@@ -1437,5 +1437,11 @@
             "name": "reason?",
             "type": "any"
         }]
+    },
+    {
+        "kind": "method",
+        "interface": "Body",
```
Comment (mhegazy) [inline inputfiles/addedTypes.json]: should have ` "flavor": "Web",` to avoid adding `FormData` in webworkers.d.ts, DOM is not available in web workers.

Comment (mhegazy) [inline inputfiles/addedTypes.json]: I believe this is what is failing the CI as well.

Comment (alfaslash) [inline inputfiles/addedTypes.json]: Thanks, I added `"flavor": "Web"`and generated *.d.ts files. Now the CI have passed.

**Pull request merged**